libcheck.sh

```diff
@@ -2,22 +2,26 @@
 
 # Report library versions used in model components of ACCESS-OM2
 # Andrew Kiss https://github.com/aekiss
+# mom_type= ACCESS-OM for default or ACCESS-OM-BGC for WOMBAT :RASF
 
 if [[ -z "${ACCESS_OM_DIR}" ]]; then
     export ACCESS_OM_DIR=$(pwd)
 fi
 if [[ -z "${LIBACCESSOM2_ROOT}" ]]; then
     export LIBACCESSOM2_ROOT=$ACCESS_OM_DIR/src/libaccessom2
 fi
+if [[ -z "${mom_type}" ]]; then
+    export mom_type=ACCESS-OM
+fi
 # if [[ -z "${OASIS_ROOT}" ]]; then
 #     export OASIS_ROOT=${ACCESS_OM_DIR}/src/oasis3-mct
 # fi
 
 declare -a libs=(openmpi netcdf)
-declare -a paths=(${ACCESS_OM_DIR}/src/mom/bin/environs.nci ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.360x300 ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.1440x1080 ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.3600x2700 ${LIBACCESSOM2_ROOT}/build_on_raijin.sh ${LIBACCESSOM2_ROOT}/build/oasis3-mct-prefix/src/oasis3-mct/util/make_dir/config.nci)
+declare -a paths=(${ACCESS_OM_DIR}/src/mom/bin/environs.nci ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.360x300 ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.1440x1080 ${ACCESS_OM_DIR}/src/cice5/bld/config.nci.auscom.3600x2700 ${LIBACCESSOM2_ROOT}/build_on_gadi.sh ${LIBACCESSOM2_ROOT}/build/oasis3-mct-prefix/src/oasis3-mct/util/make_dir/config.nci)
```

Comment:
Remove `build/oasis3-mct-prefix/src/` from `path` so it will work with oasis submodule - see https://github.com/COSIMA/access-om2/commit/ee1511552e49a429ee95d9da4d6980c2e526680f